lib/cloud/aws/labels.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package aws
+
+// Labels to add to Teleport Resources when importing them from AWS
+const (
+	labelAccountID = "account-id"
+	labelRegion    = "region"
```

Comment:
Yes, let's leave it like this for consistency reasons. However, can you please get someone's eyes on the potential bigger issue? Perhaps this is worth improving across the stack. I'm not really sure myself who should look into it, though. :/

Reply:
@smallinsky Can we please get your insight on this topic?
Was this ever an issue when onboarding databases?

Reply:
In database access we haven't had any issue with this approach and migration to custom prefixed/namespaces labels are not possible because this change will break RBAC rules. The infra labels takes precedence over our labels so even if we will add a namespace to the internal label a user can use same key and override it but this is expected behaviour.

Reply:
Thank you :pray: